core/templates/dev/head/base.html

```diff
@@ -128,12 +128,14 @@
                           <li class="dropdown pull-right">
                             <a class="dropdown-toggle oppia-navbar-profile-dropdown-toggle" data-toggle="dropdown" ng-mouseover="onMouseoverProfilePictureOrDropdown($event)" ng-mouseleave="onMouseoutProfilePictureOrDropdown($event)">
                               <div class="oppia-navbar-profile-picture-container" ng-cloak>
+                                <profile-link-image linked-name="{{username}}">
```

Comment:
I wouldn't use this directive here, actually -- I just realized that the directive is meant to be a link to the user's public profile page, and I think I would be annoyed if clicking on the top-right menu kept taking me to my profile page each time!